.github/workflows/bindings_lua.yml

```diff
@@ -46,12 +46,11 @@ jobs:
       - name: Setup lua toolchain
         run: |
           sudo apt-get update
-          sudo apt-get install -y lua-busted
+          sudo apt-get install -y lua-busted luarocks liblua5.2-dev
       - name: Setup Rust toolchain
         uses: ./.github/actions/setup
       - name: Build & Test
         working-directory: "bindings/lua"
         run: |
-          cargo build
-          mv ../../target/debug/libopendal_lua.so opendal.so
+          sudo "PATH=$PATH" luarocks make
```

Comment:
Will `sudo` be needed to run the Luarocks make command? Do you happen to know why?

Reply:
like pip, if we need to install the python package to the system environment, we need to `sudo pip install balabala`, as same as luarocks.

Reply:
`luarocks make` = `make` + `make install`

Reply:
I don't like this idea, but we can improve it in the future.